### PR TITLE
Add .editorconfig template

### PR DESCRIPTION
**Pull request by pmeinhardt**

### CODE REVIEW

File: templates/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (planktonic) [inline templates/.editorconfig]: 🎉

Comment (nickgnd) [reply templates/.editorconfig]: 💡

File: templates/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+#
+# Some of these options are also respected by Prettier.
+#
+# https://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
```
Comment (klappradla) [inline templates/.editorconfig]: 🙏

File: templates/.editorconfig [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+indent_size = 2
+indent_style = space
+insert_final_newline = true
```
Comment (klappradla) [inline templates/.editorconfig]: 🙏

**Comment by pmeinhardt** [comment]
> Not necessarily part of this PR, but one could also link to the file from the `README` of this project and add a bit of information what people are supposed to do with it.

Awesome, thanks @klappradla. I've added a couple of words to the README. Please take a look and see whether they make sense ✌️

**Comment by pmeinhardt** [comment]
I am wondering whether some bitcrowd project init tool might be cool to be able to throw these files into new projects? Maybe there's some command-line tool for snippets which pulls snippets from pre-defined repos? Or we just use `curl` to pull them from GitHub. Could be something for `bitcrowd/tools` if we decide to create that/ 🤔

**Comment by klappradla** [comment]
Interesting 💡 
When I think about super core project "defaults", I'd think about files like `README`, `gitignore` and such. `editorconfig` fits in there fine as well. I'd have seen them more in something like a "project boilerplate" defined on Github (if that exists), rather than injecting them locally.

**Comment by pmeinhardt** [comment]
Yeah 🤔 I just thought that boilerplates are super hard to maintain and they age really quickly. I was therefore wondering whether some snippet/template file collection might be a more lightweight and more granular approach. E.g. `.editorconfig` could be interesting for projects of any kind ❔

**Pull request merged**